src/nunit.analyzers/ConstraintUsage/SomeItemsConstraintUsageAnalyzer.cs

```diff
@@ -44,20 +44,20 @@ public class SomeItemsConstraintUsageAnalyzer : BaseConditionConstraintAnalyzer
             return (null, null);
         }
 
-        private bool IsLinqContains(ISymbol symbol)
+        private bool IsLinqContains(ISymbol? symbol)
         {
             return symbol is IMethodSymbol methodSymbol
                && methodSymbol.IsExtensionMethod
                && methodSymbol.Name == "Contains"
                && methodSymbol.ContainingType.GetFullMetadataName() == typeof(System.Linq.Enumerable).FullName;
         }
 
-        private bool IsCollectionContains(ISymbol symbol)
+        private bool IsCollectionContains(ISymbol? symbol)
         {
             return symbol is IMethodSymbol methodSymbol
                 && methodSymbol.Name == "Contains"
-                && (methodSymbol.IsInterfaceImplementation(typeof(ICollection<>).FullName)
-                    || methodSymbol.IsInterfaceImplementation(typeof(ICollection).FullName));
+                && (methodSymbol.IsInterfaceImplementation(typeof(ICollection<>)!.FullName!)
+                    || methodSymbol.IsInterfaceImplementation(typeof(ICollection)!.FullName!));
```

Comment:
I used `ReferenceAssemblyAnnotator` package originally, therefore FullName was annotated as nullable. Then I switched to `Nullable`, so annotation disappeared.
I decided to leave `!`, since might be needed anyway in future.

Still, suppression after `typeof` is definitely redundant, so removed it.